ci/terraform/oidc/dynamo-policies.tf

```diff
@@ -30,7 +34,7 @@ data "aws_iam_policy_document" "dynamo_user_write_policy_document" {
     ]
     resources = [
       data.aws_dynamodb_table.user_credentials_table.arn,
-      data.aws_dynamodb_table.user_profile_table.arn,
+      data.aws_dynamodb_table.common_passwords_table.arn,
```

Comment:
Is this change intentional?

Reply:
Thanks for pointing out - it was accidental on my part.